SquareLine/boards/esp32_s3_lcd_ev_board/partitions.csv

```diff
@@ -0,0 +1,6 @@
+# Name,   Type, SubType, Offset,  Size, Flags
+# Note: if you have increased the bootloader size, make sure to update the offsets to avoid overlap
+nvs,      data, nvs,     0x9000,   0x6000,
+phy_init, data, phy,     ,         0x1000,
+factory,  app,  factory, ,         3500k,
+storage,  data, spiffs,  ,         7000k,
```

Comment:
This file is not copied into SquareLine project.

Reply:
> This file is not copied into SquareLine project.

This file is copied into SquareLine project together with other files in the folder during generation. During testing it is found that a custom partition table is required even with a very basic UI.

Reply:
Are you sure? I checked generated files and it isn't there. But we can add it to `gen.py`

Reply:
I'm so sorry. I think my memory is wrong as I can't reproduce it myself! I will try to add it to gen.py. Thanks.

Reply:
Done

Reply:
Thank you for this change! Great!